src/Analysim.Web/Startup.cs

```diff
@@ -105,8 +105,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILoggerM
 
                 if (env.IsDevelopment())
                 {
-                    //spa.UseAngularCliServer(npmScript: "start");
-                    spa.UseProxyToSpaDevelopmentServer("http://localhost:4200");
+                    spa.UseAngularCliServer(npmScript: "start");
```

Comment:
@kiyros does this work, should we remove?

Reply:
it works